{
  "name": "async-file-tried",
  "version": "1.0.1",
  "description": "A try-catch wrapper around nodeâ€™s fs/promises.",
  "main": "dist/index.js",
  "types": "dist/index.d.ts",
  "type": "module",
  "scripts": {
    "test": "NODE_ENV=TEST c8 mocha",
    "coverage": "NODE_ENV=TEST c8 report --reporter=text-lcov > coverage/lcov.info"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/fwalzel/async-file-tried"
  },
  "author": "Florian Walzel",
  "license": "MIT",
  "bugs": {
    "url": "https://github.com/fwalzel/async-file-tried/issues"
  },
  "homepage": "https://github.com/fwalzel/async-file-tried#readme",
  "devDependencies": {
    "@types/node": "^20.9.5",
    "c8": "^8.0.1",
    "chai": "^4.3.6",
    "coveralls-next": "^4.2.0",
    "mocha": "^10.1.0",
    "mocha-lcov-reporter": "^1.3.0",
    "test-console": "^2.0.0"
  },
  "keywords": [
    "fs promises",
    "fs async",
    "node file system",
    "better error handling for fs",
    "try catch wrapper",
    "Typescript for fs"
  ],
  "dependencies": {
    "nyc": "^15.1.0"
  }
}
